surfsense_backend/app/agents/researcher/nodes.py

```
@@ -842,4 +842,131 @@ async def reformulate_user_query(state: State, config: RunnableConfig, writer: S
+        "chat_history": state.chat_history
+    }
+    
+    try:
+        streaming_service.only_update_terminal("✍️ Writing comprehensive answer with citations...")
+        writer({"yeild_value": streaming_service._format_annotations()})
+        
+        # Track streaming content for real-time updates
+        complete_content = ""
+        
+        # Call the QNA agent with streaming
+        async for chunk_type, chunk in qna_agent_graph.astream(qna_state, qna_config, stream_mode=["values"]):
```

Ruff (0.11.9)

Loop control variable `chunk_type` not used within loop body

Rename unused `chunk_type` to `_chunk_type`

(B007)